src/index.json

```diff
@@ -328,6 +328,51 @@
                     "summary": "An Azure CLI Extension to manage appservice resources",
                     "version": "0.2.8"
                 }
+            },
+            {
```

Comment:
How come a new extension was added instead of updating the previous extension with the new information?

Reply:
I was trying to do this to support with the new SDK vs current extensions which i don't want to break - we will eventually delete it but with some demos happening during ignite i did not want to break the current experience

Reply:
Okay, if this is the case, should we wait till after your team's ignite demo, to merge?

Reply:
@panchagnula new extension version should be ADDED to target new version of the CLI. (min version 2.0.46). I'd recommend leaving the older version of the extension for a few versions to avoid breaking.